packages/frontend/editor-ui/src/utils/stringUtils.ts

```diff
@@ -0,0 +1,16 @@
+export function splitTextBySearch(
```

Comment:
Let's add a JSDoc about what this does with a possible `@example`

```suggestion
/**
 * Description here
 */
export function splitTextBySearch(
```